ciphey/basemods/Checkers/what.py

```diff
@@ -40,7 +40,7 @@ def check(self, ctext: T) -> Optional[str]:
                 ret += f"\nClick here to view in browser [#CAE4F1][link={link}]{link}[/link][/#CAE4F1]\n"
             
             # If greppable mode is on, don't print this
-            if self.config.verbosity > 0:
+            if self.config.verbosity >= 0:

```

Comment:
this is a remnant from my xor-branch ignore it, it'll go away when xor is merged